pkg/reconciler/broker/broker.go

```diff
@@ -428,6 +435,55 @@ func (r *Reconciler) reconcileChannel(ctx context.Context, channelResourceInterf
 	return channelable, nil
 }
 
+func (r *Reconciler) reconcileBrokerChannelEventPolicies(ctx context.Context, b *eventingv1.Broker, triggerChan *duckv1.Channelable, featureFlags feature.Flags) error {
+	expected := resources.MakeEventPolicyForBackingChannel(b, triggerChan)
+
+	if featureFlags.IsOIDCAuthentication() {
+		// get the eventpolicy , create if not exists.
+		foundEP, err := r.eventPolicyLister.EventPolicies(expected.Namespace).Get(expected.Name)
+		if apierrs.IsNotFound(err) {
+			// create the EventPolicy if it doesn't exists.
+			logging.FromContext(ctx).Info("Creating EventPolicy for Broker %s", expected.Name)
```

Comment:
I'm adding a comment here, but it applies to all of the logging in this function:

I don't think these need to be logged at the `Info` level, it generally seems to be `Debug` logs. Additionally, if we are going to be using the logger everywhere, it seems to make sense to create the logger once at the start of the function and then re-use it throughout, instead of fetching it from the context every time we want to log something

Reply:
sure, will change from Info -> Debug.
On the other point, I was trying to be consistent with the rest of the code base!

Reply:
I would create a single logger object for this function if you are going to have this many log statements. In other places, since there isn't a ton of logging it might be fine to just fetch the logger everytime. But, normally we fetch the logger once when we have this much logging to do in a function

Reply:
done, PTAL!